react-native/react/actions/update.js

```diff
@@ -0,0 +1,130 @@
+/* @flow */
+
+import * as Constants from '../constants/update'
+import engine from '../engine'
+import moment from 'moment'
+
+import {updateUi} from '../constants/types/keybase_v1'
+import type {Update} from '../constants/types/flow-types'
+import type {ShowUpdateAction, RegisterUpdateListenerAction, OnCancelAction, OnSkipAction, OnSnoozeAction, OnUpdateAction, SetAlwaysUpdateAction} from '../constants/update'
+import type {Dispatch} from '../constants/types/flux'
+
+let updateResponse: ?(result: any/* UpdatePromptRes */) => void
+export const SNOOZE_TIME_SECS = 60 * 60 * 24
+
+export function registerUpdateListener (): (dispatch: Dispatch) => void {
+  updateResponse = null
+  return dispatch => {
```

Comment:
Isn't this a thunked action? redux-thunk puts `dispatch` and `getState` in.

bindActionCreator doesn't inject `dispatch` it just does
```js
bindActionCreator({foo: someActionFn}, dispatch) === {foo: dispatch(someActionFn)}
```